main.go

```diff
@@ -103,6 +105,28 @@ func main() {
 		log.Errorf("Failed to setup logger reason: %s", err)
 	}
 
+	if bot.config.NTPClient.Level != -1 {
+		bot.config.CheckNTPConfig()
+		NTPTime, errNTP := ntpclient.NTPClient(bot.config.NTPClient.Pool)
+		currentTime := time.Now()
+		if errNTP != nil {
+			log.Warnf("NTPClient failed to create: %v", errNTP)
```

Comment:
In case of this err, should skip execution the below code execution flow since the NTPTime will always differ from currentTime